src/library_webgl.js

```diff
@@ -15,6 +15,51 @@ var LibraryGL = {
   _tempFixedLengthArray__postset: 'for (var i = 0; i < 32; i++) __tempFixedLengthArray.push(new Array(i));',
   _tempFixedLengthArray: [],
 
+  _heapObjectForWebGLType: function(type) {
+    type -= 0x1400;
+#if USE_WEBGL2
+    if (type == 0x1400 - 0x1400/* GL_BYTE */) return HEAP8;
```

Comment:
For stuff like `0x1400 - 0x1400`, are you depending on closure optimizing it for you?

That seems reasonable. Another option would be to write `{{{ 0x1400 - 0x1400 }}}` and let the preprocessor do it, which would help non-closure builds too. I don't have a strong feeling either way.

Reply:
Oh nice idea - indeed this was relying on Closure, but I'll change to that form tomorrow.